types/errors.go

```diff
@@ -0,0 +1,7 @@
+package types
```

Comment:
minor nit here, because this is in a top level folder, it means in go terms it is public. In reality this should live in the internal folder.

Reply:
Done